scripts/rollup/cleanup-plugin.js

```diff
@@ -0,0 +1,46 @@
+import { transformSync as transform } from '@babel/core';
+import { createFilter } from 'rollup-pluginutils';
+
+function removeEmptyImports({ types: t }) {
+  return {
+    visitor: {
+      ImportDeclaration(path) {
+        if (path.node.specifiers.length === 0) {
+          path.remove();
+        }
+      },
+      CallExpression(path) {
+        if (
+          t.isIdentifier(path.node.callee) &&
+          path.node.callee.name === 'require' &&
+          path.parent.type === 'ExpressionStatement'
```

Comment:
This statement has been modified. Calls can also be on assignments or other statements, so the safest check is to check whether they're "freestanding", i.e. on `ExpressionStatement` nodes.